marconi-cardano-chain-index/test/Spec/Marconi/Cardano/ChainIndex/CLIInputValidation.hs

```diff
@@ -124,5 +124,4 @@ invalidCliArgWhenStartIngMarconiChainIndexTest InvalidArgTestInput{..} = do
               }
         )
       =<< Test.CLI.procFlex "marconi-cardano-chain-index" "MARCONI_CHAIN_INDEX" [invalidFlag, invalidArg]
-
-  Test.CLI.captureHandleContents (fromJust mStderr)
```

Comment:
I'm 99.999% sure this introduces a concurrency bug on aarch64-darwin, which makes stderr the empty string.